CONTRIBUTING.rst

```diff
@@ -37,6 +37,10 @@ Policy rules
 
 3. Contributions will trigger a test suite that must successfully pass before being merged (or integrated).
 
+    * The test suite can be run using a different DACCS config with ``daccs_configs_branch: branch_name`` in the PR description.
```

Comment:
Personally, I think these variables should be named `birdhouse_configs_branch` and `birdhouse_skip_ci`.

DACCS is the name of the organization that is developing Marble which uses software from Birdhouse. 

Mixing these names around can get confusing very quickly. I think that we should always use "birdhouse" to refer to elements that are specific to this repository.

Reply:
@matprov  I think mentioning daccs at all is going to lead to confusion, but if it can't be avoided then it can't be helped. I would just add an explanatory note somewhere why the name "daccs" is used here.